packages/replayio/src/utils/installation/types.ts

```diff
@@ -0,0 +1,24 @@
+export type Executable = "darwin:chromium" | "linux:chromium" | "win32:chromium";
+export type Platform = "macOS" | "linux" | "windows";
+
+// This CLI only supports Chromium for the time being
+export type Runtime = "chromium" | "gecko" | "node";
```

Comment:
It won't ever support gecko, right? 
```suggestion
export type Runtime = "chromium" | "node";
```